src/common/low_precision_transformations/src/add.cpp

```diff
@@ -237,6 +238,7 @@ bool AddTransformation::transform(TransformationContext& context, ov::pass::patt
         NetworkHelper::foldDequantization(node, fullPathIndex, defaultPrecisions);
     }
 
+    OPENVINO_DEBUG << "LPT: done: " << newAddOrSubtract;
```

Comment:
Just an idea for further improvement: we could report not only new node, but transformation type info too.

Reply:
Full file path is reported:
`benchmark_app -m model.xml -niter  1 -d GPU -data_shape "images[1,3,1024,1024]" | grep LPT`
result:
`[DEBUG] 2024-02-15T13:50:47z src/common/low_precision_transformations/src/convolution.cpp 353   LPT: done: opset1::Convolution /model.0/conv/Conv/WithoutBiases (opset1::FakeQuantize images/fq_output_0[0]:u8[?,3,?,?], opset1::Constant Constant_832465[0]:i8[80,3,3,3]) -> (f32[?,80,1..,1..])`

Reply:
Ah, that's great then 😄 Thanks for the clarification!